docs/04-guides/on-demand/store-video-asset-on-ipfs/02-api.mdx

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: api
+title: API
+---
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Video-on-Demand (VOD) API allows you to send video files to Livepeer and get them 
+ready for optimized playback. Those videos can be provided either by you (static content) 
+or your users, given your application offers an interface for them to do so.
+
+## Technical Preparation
+
+Before getting started, you will need to meet the following dependencies for
+working with the On Demand feature:
+
+- [A Livepeer Studio account](https://livepeer.studio/login)
+- Install [Node.js](https://nodejs.org/en/)
+- Access to the [API Reference](https://docs.livepeer.studio/references)
+- An [API Key](https://docs.livepeer.studio/guides/livestreaming/api-key) to
+  call the VOD API
+  - Note: Calling the API from the frontend requires an API key with CORS access
+    allowed
+
+## Caveats
+
+- Currently assets cannot be deleted from the default Livepeer Studio storage. 
+You can customize any additional storage though.
+- Only IPFS is supported as an additional storge at the moment. We are currently 
+working on support for other decentralized storages like Arweave and Storj.
+
+## Step 1: Get the $ASSET_ID of an existing asset
+Get the $ASSET_ID of an existing asset by making a request to list all existing assets in your account.
+If you haven't created an asset yet, you can follow the [upload a video asset guide](/guides/on-demand/upload-a-video-asset#api) to do so.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="javascript" label="javascript">
+
+```js
+const getAssets = await fetch("https://livepeer.studio/api/asset", {
+    method: "GET",
+    headers: {
+        Authorization: `Bearer ${process.env.LP_API_KEY}`,
+        "Content-Type": "application/json",
+    }
+});
+```
+
+</CodeBlockTabItem>
+</CodeBlockTabs>
+
+## Step 2: Export the asset to IPFS
+Once you have an $ASSET_ID, you can make a request to update the asset's storage location.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="javascript" label="javascript">
+
+```js
+const exportAssetToIPFS = await fetch("https://livepeer.studio/api/asset/{id}", {
+    method: "PATCH",
+    headers: {
+      Authorization: `Bearer ${process.env.LP_API_KEY}`,
+      "Content-Type": "application/json",
+    },
+    body: {
+        "storage": {
+            "ipfs": {}
```

Comment:
**nit:** Just as a syntax sugar, this can be `true` instead of an empty object as well :)
```suggestion
            "ipfs": true
```

Reply:
Added!